sdk/containerregistry/azure-containerregistry/tests/test_container_registry_client_async.py

```diff
@@ -612,6 +614,130 @@ async def test_get_misspell_property(self, containerregistry_endpoint):
             last_udpated_on = properties.last_udpated_on
         last_updated_on = properties.last_updated_on
         assert last_udpated_on == last_updated_on
+    
+    @acr_preparer()
+    @recorded_by_proxy_async
+    async def test_upload_oci_manifest(self, containerregistry_endpoint):
```

Comment:
Do you have tests for downloading manifests and blobs as well?

Reply:
Yes. I'm not testing them in separately like test_download_XXX(), each upload test has included `download` test, like in line631.